src/hooks/useHandleStatus.js

```diff
@@ -0,0 +1,12 @@
+import { useState } from "react";
+
+// 모달 on/off
+export const useHandleStatus = () => {
+  const [isStatus, SetIsStatus] = useState(false);
+
+  const handleStatus = () => {
+    SetIsStatus(!isStatus);
+    document.body.style.overflow = "unset";
+  };
+  return { isStatus, handleStatus };
+};
```

Comment:
- camelCase 주의해주세요!
- 그리고 state명과 hook 이름이 좀 더 명확하면 좋을 것 같습니다.
지금은 true/false를 바꿔주는 내용이기 때문에 useToggle같은 명칭이 더 정확할 것 같네요!
- document body에 styling하는 부분은 실제 해당 부분을 사용하는 로직에서 추가하면 좋을 것 같습니다.
- return하는 형태를 객체가 아닌 배열로 하면 실제로 해당 hook을 사용하는 부분에서 알맞게 대입해 사용할 수 있을 것 같습니다.
```suggestion
export const useToggle = (initialState = false) => {
  const [isToggled, setIsToggled] = useState(initialState);

  const handleIsToggled = () => {
    setIsToggled(!isToggled);
  };
  
  return [ isToggled, handleIsToggled ];
};
```

사용 시 - `const [isModal, handleModal] = useToggle()`

Reply:
리뷰주신 부분 반영하다가 state가 업데이트 되지 않는 문제를 발견했는데... 업데이트 함수 관련 문제였습니다.
```javascript
setIsToggled(value => !value);
```
이렇게 작성해야 즉시 업데이트 되었어요!